Source/WebCore/html/HTMLInputElement.idl

```diff
@@ -25,10 +25,12 @@
     Exposed=Window
 ] interface HTMLInputElement : HTMLElement {
     [CEReactions=NotNeeded, Reflect] attribute DOMString accept;
+    [EnabledBySetting=InputTypeColorEnhancementsEnabled, CEReactions=NotNeeded, Reflect] attribute boolean alpha;
```

Comment:
I need to add `Conditional=INPUT_TYPE_COLOR` here as the binding layer is not that smart.